core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/deploysemantics/PhysicalLink.scala

```diff
@@ -118,11 +23,9 @@ class PhysicalLink(
     val fromPort: Int,
```

Comment:
The usage of `fromOp` and `toOp` are just getting their operator ID. As the partitioning is being moved to the config. Can we merge PhysicalLinkIdentity and PhysicalLink into one class?

Reply:
yes we will merge those two. will modify it in the future PR.